hazelcast/src/test/java/com/hazelcast/quorum/executor/ExecutorQuorumWriteTest.java

```diff
@@ -453,19 +455,14 @@ public void invokeAll_noQuorum() throws Exception {
     }
 
     @Test
-    public void invokeAll_timeout_quorum() throws Exception {
-        try {
-            expectQuorumException(exec(0).invokeAll(Arrays.<Callable<Object>>asList(callable(), callable()), 10l, TimeUnit.SECONDS));
-        } catch (UnsupportedOperationException ex) {
-        }
+    public void invokeAll_timeout_quorum_short_timeout() throws Exception {
+        List<Future<?>> futures = exec(0).invokeAll(Arrays.<Callable<Object>>asList(callable(), callable()), 10l, TimeUnit.SECONDS);
+        assertAllowedException(futures, CancellationException.class);
```

Comment:
A long enough timeout doesn't mean a long (default) test execution, right? It would just not throw a `CancellationException` if the test is paused due to a hiccup.

If that's the case, I would add a second test case (maybe with a comment to explain the exception situation).

Reply:
can do

Reply:
@Donnerbart: I pushed an update with a longer timeout